Resources/Migrations/eeMigration.yml

```diff
@@ -4,4 +4,57 @@ CrateFunLizardPlushieBulk: CrateFunToyBox
 
 # 2024-08-27
 Oracle: OracleSpawner
-SophicScribe: SophicScribeSpawner
\ No newline at end of file
+SophicScribe: SophicScribeSpawner
+
+# 2024-09-06
+ClothingUniformJumpsuitProsecutor: null
+ClothingUniformJumpsuitClerk: null
+ClothingUniformJumpsuitChiefJustice: null
+ClothingUniformJumpskirtChiefJustice: null
+ClothingUniformJumpsuitChiefJusticeWhite: null
+ClothingUniformJumpsuitChiefJusticeFormal: null
+ClothingOuterChiefJustice: null
+ChiefJusticePDA: null
+ChiefJusticeIDCard: null
+LockerChiefJustice: null
+LockerChiefJusticeFilled: null
+SpawnPointChiefJustice: null
+DefaultStationBeaconJustice: null
+DefaultStationBeaconChiefJustice: null
+DefaultStationBeaconProsecutor: null
+RubberStampChiefJustice: null
+AirlockJustice: Airlock
+AirlockJusticeGlass: AirlockGlass
+AirlockChiefJusticeLocked: AirlockLawyerLocked
+AirlockProsecutorLocked: AirlockLawyerLocked
+AirlockChiefJusticeGlassLocked: AirlockLawyerGlassLocked
+AirlockProsecutorGlassLocked: AirlockLawyerGlassLocked
+AirlockJusticeLocked: AirlockLawyerLocked
+AirlockMaintChiefJusticeLocked: AirlockLawyerLocked
+AirlockMaintJusticeLocked: AirlockLawyerLocked
+AirlockMaintProsecutorLocked: AirlockLawyerLocked
+WindoorSecureChiefJusticeLocked: WindoorSecureLawyerLocked
+WindoorSecureProsecutorLocked: WindoorSecureLawyerLocked
+WindoorSecureJusticeLocked: WindoorSecureLawyerLocked
+DoorElectronicsChiefJustice: DoorElectronicsLawyer
+DoorElectronicsJustice: DoorElectronicsLawyer
+DoorElectronicsProsecutor: DoorElectronicsLawyer
+ClothingNeckProsecutorbadge: null
+ProsecutorPDA: null
+ClothingUniformJumpskirtProsecutor: null
+BoxPDAJustice: null
+SpawnPointProsecutor: null
+ClothingUniformJumpskirtClerk: null
+SpawnPointClerk: null
+ProsecutorIDCard: null
+LockerClerk: null
+LockerclerkFilled: null
+ClerkPDA: null
+ClerkIDCard: null
+RubberStampNotary: null
+EncryptionKeyJustice: null
+ClothingHeadsetJustice: null
+DoorRemoteJustice: null
+ClothingHeadHatCJToque: null
+SignDirectionalJuistice: null
+PaperStationWarrant: null
```

Comment:
```suggestion
ChiefJusticePDA: null
ChiefJusticeIDCard: null
LockerChiefJustice: null
LockerChiefJusticeFilled: null
SpawnPointChiefJustice: null
DefaultStationBeaconJustice: null
DefaultStationBeaconChiefJustice: null
DefaultStationBeaconProsecutor: null
RubberStampChiefJustice: null
AirlockJustice: Airlock
AirlockJusticeGlass: AirlockGlass
AirlockChiefJusticeLocked: AirlockLawyerLocked
AirlockProsecutorLocked: AirlockLawyerLocked
AirlockChiefJusticeGlassLocked: AirlockLawyerGlassLocked
AirlockProsecutorGlassLocked: AirlockLawyerGlassLocked
AirlockJusticeLocked: AirlockLawyerLocked
AirlockMaintChiefJusticeLocked: AirlockLawyerLocked
AirlockMaintJusticeLocked: AirlockLawyerLocked
AirlockMaintProsecutorLocked: AirlockLawyerLocked
WindoorSecureChiefJusticeLocked: WindoorSecureLawyerLocked
WindoorSecureProsecutorLocked: WindoorSecureLawyerLocked
WindoorSecureJusticeLocked: WindoorSecureLawyerLocked
DoorElectronicsChiefJustice: DoorElectronicsLawyer
DoorElectronicsJustice: DoorElectronicsLawyer
DoorElectronicsProsecutor: DoorElectronicsLawyer
ProsecutorPDA: null
BoxPDAJustice: null
SpawnPointProsecutor: null
SpawnPointClerk: null
ProsecutorIDCard: null
LockerClerk: null
LockerclerkFilled: null
ClerkPDA: null
ClerkIDCard: null
RubberStampNotary: null
EncryptionKeyJustice: null
ClothingHeadsetJustice: null
DoorRemoteJustice: null
SignDirectionalJuistice: null
PaperStationWarrant: null
```
Okay, I've put in a list of all the items I'd like for us to keep, but to instead just be Lawyer Loadout Items.